### ISSUE TITLE
[Bug] "Switch unit" module do not return you to Zeus until you are dead

### ISSUE THREAD

**Issue by unhappytroll** [ArmaAchilles/AresModAchillesExpansion#177]
**Arma 3 Version:** `1.76` (stable)  x64 client
**CBA Version:** `3.4.1` (stable)  
**AresModAchillesExpansion Version:** `0.0.9c` (stable)  

**Mods:**
```
- CBA_A3
- AresModAchillesExpansion
```

**Description:**
Module  "Switch unit" will not return you to Zeus interface by Zeus key. You will be returned into it only after death. Does not depends on the modset, does not depends if there is only Zeus camera (instead of player model) in the mission.

**Steps to reproduce:**
1. Place youself and Zeus module into mission in Eden.
2. Play it as Singleplayer or Multiplayer from Eden.
3. Enter Zeus interface.
4. Place a unit.
5. Put a "Switch unit" module on placed unit or Alt+Double left click on it.
6. Push Zeus button -there goes ping. At this point you will not be returned into Zeus until you alive.

**Where did the issue occur?**
- Editor (Singleplayer) / Editor (Multiplayer)
Did not tried on any other modes yet.

**RPT log file:**
No RPT, because there is no crash occurs.

**Comment by unhappytroll** [comment]
RPt added, but there is nothing to see in it looks like.
https://pastebin.com/J9uwU0Xd

**Comment by unhappytroll** [comment]
https://pastebin.com/dTsPaj1K
This time it releases me into Zeus module after grenade suicide attempt - but unit was not dead, though it should be.

**Comment by unhappytroll** [comment]
Confirmed on dedicated. 
RPT: https://pastebin.com/fKHzFCRp

**Comment by CreepPork** [comment]
There are some script errors in your mission, perhaps that is the cause. I still cannot reproduce the issue.

**Comment by unhappytroll** [comment]
There is absolutely nothing in my mission, except Zeus player and Zeus module.

**Comment by CreepPork** [comment]
I did everything as you did in your linked video but it still doesn't want to break. I tested with 0.0.9c and 0.1.0 and no luck.

**Comment by unhappytroll** [comment]
This is why it is very strange. How it should be at least? And can it be dependent on key remapping?

**Comment by CreepPork** [comment]
It shouldn't change anything, did you rebind the Zeus key?

**Comment by unhappytroll** [comment]
Yes, to Right Ctrl+Y. To remove conflict with TFAR.

**Comment by CreepPork** [comment]
Yes, I changed the keybinding to your's and now it doesn't want to reopen.

**Comment by unhappytroll** [comment]
Ok, can confirm that this is an Arma issue, because after changing keybind into key without modifiers it works. Case closed.

**Comment by CreepPork** [comment]
This is not a BIS fault, but our issue.

**Comment by CreepPork** [comment]
After some looking into this issue when the keybinding has a modifier key attached to it, there is another issue. When you have the Zeus interface set to locked, you can still exit it with the new keybinding.

The biggest issue is that the `actionName` doesn't support modifier keys.
